src/components/exercise-identifier-link.cjsx

```diff
@@ -23,8 +23,5 @@ ExerciseIdentifierLink = React.createClass
       <span className='exercise-identifier-link'>
         ID# {@props.exerciseId} | <a target="_blank" href={url}>Report an error</a>
       </span>
-      <p>
-        { _.map(@props.related_content, @buildLabel) }
-      </p>
     </div>
```

Comment:
Although the `buildLabel` method really should be wrapped in a element of some sort with a `key` prop.  Otherwise we're going to have React warnings, since it's using `_.map` here